src/args.rs

```diff
@@ -26,6 +26,20 @@ pub struct CliArgs {
     #[arg(value_hint = ValueHint::AnyPath, env = "MINISERVE_PATH")]
     pub path: Option<PathBuf>,
 
+    /// The path to where file uploads will be written to before being moved to their
+    /// correct location. It's wise to make sure that this directory will be written to
+    /// disk and not into memory.
+    ///
+    /// This value will only be used **IF** file uploading is enabled. If this option is
+    /// not set, the operating system default temporary directory will be used.
+    #[arg(
+        long = "temp-directory",
+        value_hint = ValueHint::FilePath,
+        requires = "allowed_upload_dir",
+        env = "MINISERVER_TEMP_UPLOAD_DIRECTORY")
+    ]
+    pub temp_upload_directory: Option<PathBuf>,
```

Comment:
A thought I had about this: If the dir is chosen so that it is in a served path, should we serve it? Or hide (and 403 any direct file hits) it or show it in some special way?

Reply:
Personally, I would just show the directory and the file uploads that are happening, because if someone decides to set the upload directory to something visible, they are probably doing so on purpose.

I think that this should be addressed _if_ someone posts an issue about it.

Reply:
That's a fair point!